### PR TITLE
[nav] Remove Radio Repeater code from nav #871

### LINKED ISSUE #869: [nav] Remove Radio repeater code
1/15 integration test revealed that the RR code in `jetson/nav` was causing segfaults in communicating with `base_station`. Making an issue to go along with future PR.

### PR DESCRIPTION
**Pull request by AjaySum**
resolves #869 

Removed all radio repeater code, which fixes segfault issues when running `jetson/nav` with `base_station`.

### CODE REVIEW

File: jetson/nav/rover.hpp [hunk 1]
```diff
@@ -113,14 +106,16 @@ class Rover
...
-        Target& target();
+        Target& target1();
 
         Target& target2();
 
         RadioSignalStrength& radio();
 
         unsigned getPathTargets();
 
+        RadioSignalStrength mSignal;
```
Comment (niwhsa9) [inline jetson/nav/rover.hpp:117]: needed?

**Review by niwhsa9 — APPROVED** [review]

File: jetson/nav/rover.hpp (hunk 1 above)
Comment (zwgold) [reply jetson/nav/rover.hpp:117]: good call, wanted to keep it in but in emergencies we can probs just revert the commit/copy back in code

**Comment by zwgold** [comment]
Code compiles and runs the same as before, this can be merged.

**Review by zwgold — DISMISSED** [review]

File: jetson/nav/rover.cpp [hunk 2]
```diff
@@ -52,17 +52,13 @@ Odometry& Rover::RoverStatus::odometry()
 // Gets a reference to the rover's first target's current information.
 Target& Rover::RoverStatus::target()
 {
-    return mTarget1;
+    return mTarget;
```
Comment (cgiger00) [inline jetson/nav/rover.cpp:55]: how does this relate to the repeater?

File: jetson/nav/rover.cpp [hunk 3]
```diff
@@ -90,9 +86,8 @@ Rover::RoverStatus& Rover::RoverStatus::operator=( Rover::RoverStatus& newRoverS
     }
     mObstacle = newRoverStatus.obstacle();
     mOdometry = newRoverStatus.odometry();
-    mTarget1 = newRoverStatus.target();
+    mTarget = newRoverStatus.target();
```
Comment (cgiger00) [inline jetson/nav/rover.cpp:89]: again...the repeater?

**Review by cgiger00 — COMMENTED** [review]
Looks ok, mostly confused about why there are the target1 to target changes in here with the radio repeater clean up

**Review by zwgold — APPROVED** [review]
lgtm

**Review by cgiger00 — APPROVED** [review]

**Pull request merged**